azkaban-common/src/main/java/azkaban/scheduler/Schedule.java

```diff
@@ -63,7 +68,7 @@ public Schedule(int scheduleId, int projectId, String projectName,
 
     this(scheduleId, projectId, projectName, flowName, status, firstSchedTime,
         timezone, period, lastModifyTime, nextExecTime, submitTime, submitUser,
-        null, null);
+        null, null, null);
```

Comment:
There are too many parameters. Maybe using a builder would be useful. You can refactor it in a separate commit if you decide to do so.